wagtail/admin/templates/wagtailadmin/shared/breadcrumbs.html

```diff
@@ -5,7 +5,7 @@
     `pages` - A list of wagtail page objects
     `trailing_breadcrumb_title` (string?) - use this for a non linkable last breadcrumb
 {% endcomment %}
-{% with breadcrumb_link_classes='w-flex w-items-center w-h-full w-text-primary w-px-0.5 w-text-14 w-no-underline w-outline-offset-inside hover:w-underline hover:w-text-primary w-h-full' breadcrumb_item_classes='w-h-full w-flex w-items-center w-overflow-hidden w-transition w-duration-300 w-whitespace-nowrap w-flex-shrink-0 w-font-bold' icon_classes='w-w-4 w-h-4 w-ml-3' %}
+{% with breadcrumb_link_classes='w-flex w-items-center w-h-full w-text-primary w-px-0.5 w-text-14 w-no-underline w-outline-offset-inside hover:w-underline hover:w-text-primary w-h-full' breadcrumb_item_classes='w-h-full w-flex w-items-center w-overflow-hidden w-transition w-duration-300 w-whitespace-nowrap w-flex-shrink-0 w-font-bold' icon_classes='w-w-4 w-h-4 w-ml-3' trailing_breadcrumb_title=trailing_breadcrumb_title %}
```

Comment:
not sure why you need this change, the `trailing_breadcrumb_title` should already be in context and using `with` here is basically saying set the context variable `trailing_breadcrumb_title` to the value `trailing_breadcrumb_title` from context.

Reply:
Oh right. I started working on it yesterday but paused in the middle to wait for existing PRs to merge to avoid conflicts.

Glad you like the name:)